tests/test_registrations/test_views.py

```diff
@@ -432,6 +432,19 @@ def test_delete_draft_registration_registered(self, mock_register_draft):
         res = self.app.delete(url, auth=self.user.auth, expect_errors=True)
         assert_equal(res.status_code, http.FORBIDDEN)
 
+    @mock.patch('website.archiver.tasks.archive')
+    def testn_delete_draft_registration_approved_and_registration_deleted(self, mock_register_draft):
```

Comment:
The `testn` might prevent this test from being discovered by `nose`. I'll fix this when I merge.